### ISSUE TITLE
GENERAL DIGITAL PIN

### ISSUE THREAD

**Issue by Rydec** [noisymime/speeduino#414]
IT would glad if there is general IO pin that can switch on or off  in tunerstudio without any conditional input. Basically it will be use to enable pullup for hall effect sensor without opening the box. Or maybe turn on/off bluetooth module in the box OR change voltage of ignition driver or stepper driver with relay.

**Comment by morrisman1** [comment]
agreed, to achieve this on my setup I just set a condition, when MAP > 1
